src/connector.js

```diff
@@ -790,6 +795,12 @@ function initEngine() {
 	}
 }
 
+function clearCoveoAnalyticsHistory(){
+	const storageKey = '__coveo.analytics.history'
+	document.cookie =`${storageKey}=; expires=; domain=; path=/; SameSite=Lax;`
+	localStorage.removeItem(storageKey);
```

Comment:
Could this be put in an IF statement so that it gets deleted **only if** it exists? Just for efficiency purposes.